packages/core/src/docdb/utils.ts

```diff
@@ -128,3 +128,22 @@ export function validateInstanceName(name: string): string | undefined {
 
     return undefined
 }
+
+/**
+ * Format for rendering readable dates as YYYY-MM-DD
+ */
+export function formatDate(date: Date = new Date()): string {
+    const year = date.getFullYear()
+    const month = (date.getMonth() + 1).toString().padStart(2, '0')
+    const day = date.getDate().toString().padStart(2, '0')
+    return `${year}-${month}-${day}`
+}
+
+/**
+ * Format for rendering the time as HH:mm
+ */
+export function formatTime(date: Date = new Date()): string {
+    const hours = date.getHours().toString().padStart(2, '0')
+    const minutes = date.getMinutes().toString().padStart(2, '0')
+    return `${hours}${minutes}`
```

Comment:
Two potential things here.

1. Hours and minutes are being combined with no separator (i.e. `1231` for `12:31 PM`). Should we have a `-` or something?
2. Is minute fine grained enough to prevent collisions in snapshot ID creation?

Reply:
1. We could go either way.
2. It should be, as it takes longer than a minute to make a snapshot.

Reply:
I have no strong preference on #1. Feel free to go whatever direction you see fit.